plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkIndexByScrollRequestBuilder.java

```diff
@@ -0,0 +1,24 @@
+package org.elasticsearch.plugin.reindex;
```

Comment:
Sure! This used to "just work" in eclipse. I wonder if this is something that gradle dropped accidentally....